src/state/apis/limit-orders/limitOrderApi.ts

```diff
@@ -1,40 +1,50 @@
-import { createApi } from '@reduxjs/toolkit/dist/query/react'
-import type { AssetId, AssetReference } from '@shapeshiftoss/caip'
-import { type ChainId, fromAssetId } from '@shapeshiftoss/caip'
-import type { CowSwapQuoteError } from '@shapeshiftoss/swapper'
-import {
-  CoWSwapOrderKind,
-  CowSwapQuoteErrorType,
-  CoWSwapSellTokenSource,
-  CoWSwapSigningScheme,
-  getCowswapNetwork,
-  TradeQuoteError,
-} from '@shapeshiftoss/swapper'
+import { createApi, fakeBaseQuery } from '@reduxjs/toolkit/dist/query/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { type ChainId, fromAssetId, fromChainId } from '@shapeshiftoss/caip'
+import type { SignTypedDataInput } from '@shapeshiftoss/chain-adapters'
+import { toAddressNList } from '@shapeshiftoss/chain-adapters'
+import type { ETHSignTypedData, HDWallet } from '@shapeshiftoss/hdwallet-core'
+import type { CowSwapError } from '@shapeshiftoss/swapper'
+import { COW_SWAP_SETTLEMENT_ADDRESS, getCowswapNetwork } from '@shapeshiftoss/swapper'
 import { COW_SWAP_NATIVE_ASSET_MARKER_ADDRESS } from '@shapeshiftoss/swapper/dist/swappers/CowSwapper/utils/constants'
 import {
+  domain,
   getAffiliateAppDataFragmentByChainId,
   getFullAppData,
+  getSignTypeDataPayload,
 } from '@shapeshiftoss/swapper/dist/swappers/CowSwapper/utils/helpers/helpers'
 import { isNativeEvmAsset } from '@shapeshiftoss/swapper/dist/swappers/utils/helpers/helpers'
+import type {
+  GetOrdersRequest,
+  Order,
+  OrderCancellation,
+  OrderId,
+  OrderStatus,
+  QuoteId,
+  Trade,
+} from '@shapeshiftoss/types/dist/cowSwap'
+import {
+  type OrderCreation,
+  type OrderQuoteRequest,
+  type OrderQuoteResponse,
```

Comment:
nitpick: seems like we always import types seperated from actual JS

Reply:
This has come up a couple of times. I'm relying on the linter here, so if we want to set a rule we should do it there 😅